tests/serverpod_test_server/lib/src/endpoints/server_only_scoped_field_model_child.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:async';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+
+class ServerOnlyScopedFieldChildModelEndpoint extends Endpoint {
+  Future<ScopeServerOnlyField> getProtocolField(
```

Comment:
Suggestion: This endpoint doesn't return `ScopeServerOnlyFieldChild` which is what we want to test.

Please change the return type to catch this specific case.